docs/howto/InitialSetup.md

```diff
@@ -0,0 +1,58 @@
+
```

Comment:
Make sure that this is not super redundant with other guides. I feel this is super redundant. We have protege setup guides, id ranges file guide. Lets make sure these are all properly merged.

Reply:
Oh it is, sorry, I moved this out into separate documents and forgot to delete this